components/service/glean/src/main/java/mozilla/components/service/glean/storages/StringListsStorageEngine.kt

```diff
@@ -109,7 +123,19 @@ internal open class StringListsStorageEngineImplementation(
         metricData: CommonMetricData,
         value: List<String>
     ) {
-        if (value.count() > MAX_LIST_LENGTH_VALUE) {
+        val stringList = value.map {
+            if (it.length > MAX_STRING_LENGTH) {
+                recordError(
+                    metricData,
+                    ErrorType.InvalidValue,
+                    "${metricData.identifier} - string too long ${it.length} > $MAX_STRING_LENGTH",
```

Comment:
We don't need to include `metricData.ifentifier` in the message: `recordError` will automatically add it.

Reply:
Dang, I thought I got all of those...  good catch!